tests/test/faaslet/test_pool.cpp

```diff
@@ -34,11 +34,9 @@ static void tearDown()
     cleanSystem();
 }
 
-TEST_CASE("Test worker initially pre-warmed", "[faaslet]")
+TEST_CASE("Test faaslet not initially bound", "[faaslet]")
 {
     setUp();
-
-    FaasletPool pool(1);
```

Comment:
In the past these tests had to create an instance of `FaasletPool`, but don't need to any more so I've removed them